pkg/analysis/passes/manifest/manifest.go

```diff
@@ -215,11 +215,11 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return b, nil
 }
 
-// parseManifestFile parses the manifest file and returns a ManifestFile struct
+// parseManifestFile parses the manifest file and returns a File struct
 // it does not verify the signature only returns the content
-func parseManifestFile(b []byte) (ManifestFile, error) {
+func parseManifestFile(b []byte) (File, error) {
```

Comment:
IMO this should remain called "ManifestFile"

Reply:
The package is already called `manifest`, so we should avoid repeating it: https://go.dev/blog/package-names#naming-package-contents

I'd like to keep it like this

```go
manifest.ManifestFile
```

:point_down: 

```go
manifest.File
```